lib/adyen/client.rb

```diff
@@ -60,6 +60,9 @@ def service_url_base(service)
         when "BalancePlatform" 
           url = "https://balanceplatform-api-#{@env}.adyen.com/bcl"
           supports_live_url_prefix = false
+        when "Transfers"
+          url = "https://balanceplatform-api-test.adyen.com/btl"
```

Comment:
Looks like we need a variable here: 
```suggestion
          url = "https://balanceplatform-api-#{@env}.adyen.com/btl"
```

Reply:
Oops, missed it this time.